machine-translation/deployment/main.py

```diff
@@ -0,0 +1,74 @@
+"""Kserve inference script."""
+
+import argparse
+from kserve import (
+    Model,
+    ModelServer,
+    model_server,
+    InferRequest,
+    InferOutput,
+    InferResponse,
+)
+from kserve.utils.utils import generate_uuid
+from transformers import AutoTokenizer
+from transformers import TFAutoModelForSeq2SeqLM
+
+class MachineTranslation(Model):
+    """Kserve inference implementation of model."""
+
+    def __init__(self, name: str, protocol: str):
+        """Initialise model."""
+        super().__init__(name)
+        self.model = None
+        self.tokenizer = None
+        self.protocol = protocol
+        self.ready = False
+        self.load()
+
+    def load(self):
+        """Reconstitute model from disk."""
+        self.tokenizer = AutoTokenizer.from_pretrained("tf_saved_model")
+        self.model = TFAutoModelForSeq2SeqLM.from_pretrained("tf_saved_model/")
+        self.ready = True
+
+
+    def predict(self, payload: InferRequest) -> InferResponse:
```

Comment:
```suggestion
    def predict(self, payload: InferRequest, *args, **kwargs) -> InferResponse:
```
@Verosha0803 when we actually run this during local testing and in Highwind, there are some other arguments passed to this function, like `headers`. We can add `*args, **kwargs` so that these don't cause an error during inference.

Reply:
Updated, thanks for picking this up!